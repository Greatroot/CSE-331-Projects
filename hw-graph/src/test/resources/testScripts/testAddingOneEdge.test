# Test adding one edge | should add the edge

CreateGraph graph
AddNode graph n1
AddNode graph n2
AddEdge graph n1 n2 e1