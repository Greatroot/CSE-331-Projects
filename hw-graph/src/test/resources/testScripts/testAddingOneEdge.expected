# Test adding one edge | should add the edge

created graph graph
added node n1 to graph
added node n2 to graph
added edge e1 from n1 to n2 in graph